packages/devtools_app/lib/src/memory/memory_allocation_table_view.dart

```diff
@@ -0,0 +1,89 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+import '../auto_dispose_mixin.dart';
+import '../table.dart';
+import '../table_data.dart';
+import '../utils.dart';
+import 'memory_allocation_table_data.dart';
+import 'memory_controller.dart';
+import 'memory_protocol.dart';
+
+class AllocationTableView extends StatefulWidget {
+  @override
+  AllocationTableViewState createState() => AllocationTableViewState();
+}
+
+/// Table of the fields of an instance (type, name and value).
+class AllocationTableViewState extends State<AllocationTableView>
+    with AutoDisposeMixin {
+  MemoryController controller;
+
+  final List<ColumnData<ClassHeapDetailStats>> columns = [];
+
+  @override
+  void initState() {
+    setupColumns();
+
+    super.initState();
```

Comment:
@pq here is an example of a lint that could help avoid a case that trips me up all the time.
https://api.flutter.dev/flutter/widgets/State/initState.html
calls to `super.initState()` should occur at the top op of the initState method by convention. Similarly, calls to super.dispose() should occur at the bottom.

Reply:
Done.